File: c/uptonseries.c
#include <stdio.h>

int main(void)
{
  int Count, Multiple, Counter, Term = 1;
  printf("Enter a number upto which you want to print the terms in a series:");
  scanf("%d", &Count);
  printf("The terms in the series upto %d are", Count);
  for(Multiple = 1; Term <= Count; Multiple++)
  {
      Term = 1;
      for(Counter = 1; Counter <= Multiple; Counter++)
      {
         Term = Term * Multiple;
      }
      if(Term <= Count)
      {
         printf(" %d", Term);
      }
  }
  printf(".\n");
  return 0;
}

File: python/uptonseries.py
Count = int(input("Enter a number upto which you want to print the series:"))
Multiple = 1
Term = 1
print('The terms in the series upto {0} are' .format(Count), end = "")
while(Term <= Count):
  Term = 1
  for Counter in range(1, Multiple + 1):
    Term = Term * Multiple
  if(Term <= Count):
    print("", Term, end = "")
  Multiple = Multiple + 1
print(".")
